### PR TITLE
[FIX] base: fix traceback when exporting the translation without model

### PR DESCRIPTION
**Pull request by alsh-odoo**
Currently, a traceback is occurring when the user tries to export a translation 
without selecting the model when the export type is `model`

To reproduce this issue:

1) Try to export a translation without a model with the export type as `model`

Error:- 
```
KeyError: False
```

We can see that the model is not required when exporting a translation when the export type is `model`.

This leads to a Keyerror from the below line

https://github.com/odoo/odoo/blob/ce786882fe852786441ab86d9311aecc143cf57d/odoo/addons/base/wizard/base_export_language.py#L43-L44

Making the model as required in a stable version if the export type is `model` in XML is not
stable friendly.

So raising a `Usererror` will resolve this issue in stable.

Note:- Will make the model as required if the export_type is `model` in master in xml

sentry-5686584762

### CODE REVIEW

**Comment by alsh-odoo** [comment]
Ola @ryv-odoo :wave: 

Requesting your review on this small PR

Thank You :slightly_smiling_face:

**Review by ryv-odoo — APPROVED** [review]

**Comment by alsh-odoo** [comment]
@ryv-odoo :wave: 

Requested changes are done

File: odoo/addons/base/wizard/base_export_language.py [hunk 1]
```diff
@@ -41,6 +42,8 @@ def act_getfile(self):
 
         with contextlib.closing(io.BytesIO()) as buf:
             if this.export_type == 'model':
+                if not this.model_name:
+                    raise UserError(_("Model is required to export the translation"))
```
Comment (ryv-odoo) [inline odoo/addons/base/wizard/base_export_language.py:46]: Isn't it better to make the field model_id required in the view when the `export_type == 'model'` ? :thinking: 
Also it avoids some work to the translation team.

**Review by ryv-odoo — CHANGES_REQUESTED** [review]

**Comment by ryv-odoo** [comment]
@robodoo r+
Thx

**Pull request closed without merging**